samples/bluetooth/peripheral_dis/prj.conf

```diff
@@ -1,5 +1,28 @@
 CONFIG_BT=y
 CONFIG_BT_PERIPHERAL=y
+
+
```

Comment:
OK

Reply:
No need for two empty lines, just use one